extensions/wikia/AdEngine/js/spec/lookup/rubicon/rubiconVulcan.spec.js

```diff
@@ -183,16 +191,16 @@ describe('ext.wikia.adEngine.lookup.rubicon.rubiconVulcan', function () {
 		});
 	});
 
-	it('Returns proper tier format based on response', function () {
-		var vulcan = getVulcan();
-
-		mocks.vulcanResponse.cpm = 0.23;
-		vulcan.call();
-
-		expect(vulcan.getSlotParams('INCONTENT_LEADERBOARD')).toEqual({
-			'rpfl_video': '203_tier0020'
-		});
-	});
+	//it('Returns proper tier format based on response', function () {
+	//	var vulcan = getVulcan();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)